packages/api/src/blog.js

```diff
@@ -0,0 +1,16 @@
+import { JSONResponse } from './utils/json-response.js'
+
+/**
+ * @param {import('./user').AuthenticatedRequest} request
+ * @param {import('./env').Env} env
+ * @param {import('./index').Ctx} ctx
+ * @return {Promise<JSONResponse>}
+ */
+export async function blogSubscriptionCreate (request, env, ctx) {
```

Comment:
can this be removed and `lib/api.js` used instead?

Reply:
I might be misunderstanding this so bear with me:

I think `lib/api.js` is a client side wrapper around the API call that goes into the `blogSubscriptionCreate` API

so they're the same thing, one i just a client side wrapper around a server side endpoint

Reply:
ok.  so it looks like this is the only `blogSubscriptionCreate` that's being used.  i don't see the one you added in the `website` package imported from anywhere.

Reply:
It isn't being used anywhere yet.  The frontend is being done by a different group.  I think it's Potato.  @JeffLowe do you recall which team it is?

Reply:
Talked with @jsdevel offline.  It sounds like he'd prefer I remove the `lib/api.js` bit so it's just a backend stub.  Which seems fine.